gui_qt/app.py

```
@@ -638,6 +632,22 @@ def _build_global_project_bar(self) -> QFrame:
+
+        # Project-level prep lives with directory switch (not per-task action row).
+        self.doctor_btn = QPushButton("环境检查")
+        self.doctor_btn.setObjectName("secondary_btn")
+        self.doctor_btn.setToolTip("环境检查 (Ctrl+D)")
+        self.doctor_btn.clicked.connect(self._on_run_doctor)
+        self.global_project_actions.add_widget(self.doctor_btn, min_width=88)
+
+        self.bootstrap_work_btn = QPushButton("准备工作目录")
+        self.bootstrap_work_btn.setObjectName("secondary_btn")
+        self.bootstrap_work_btn.setToolTip(
+            "从 original/game 复制到 work/game（若已存在则按 CLI 规则跳过或更新）。"
```

Ruff (0.15.20)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)